src/cpp/core/ConfigProfile.cpp

```diff
@@ -88,7 +88,7 @@ Error ConfigProfile::parseString(const std::string& profileStr)
    for (const ptree::value_type& child : profileTree)
    {
       boost::optional<Level> matchingLevel;
-      for (const Level& level : sections_)
+      for (Level level : sections_)
```

Comment:
This is a copy instead of a reference because it's an enum type (and therefore just as fast/memory efficient to copy as to copy the reference of it)?

Reply:
Right, exactly.